linode/client.go

```diff
@@ -0,0 +1,136 @@
+package linode
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/libdns/libdns"
+	"github.com/linode/linodego"
+)
+
+func (p *Provider) init(ctx context.Context) {
+	p.client = linodego.NewClient(http.DefaultClient)
```

Comment:
This probably only needs to happen once (and should be synced to be thread-safe; use a mutex, probably)

Reply:
I'll update it. I want to add some unit tests too. Thanks for the feedback.